galasa-parent/settings.gradle

```diff
@@ -22,6 +22,7 @@ include 'dev.galasa.framework.metrics'
 include 'dev.galasa.framework.api'
 include 'dev.galasa.framework.api.authentication'
 include 'dev.galasa.framework.api.bootstrap'
+include 'dev.galasa.framework.api.client'
```

Comment:
We likely need `dev.galasa.framework.api.client` in the framework's release.yaml as well

Reply:
Ok. Added.